gallery/src/GalleryNav/GalleryNavIcons.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+export const GalleryNavIconCSS3 = () => (
+  <svg className="gallery-nav-icon gallery-nav-icon--css3"
+       width="20"
+       height="20"
+       viewBox="0 0 20 20"
+       fill="none"
```

Comment:
This seems unnecessary; it's overridden on every single child.  I'd say rather than setting `fill=currentColor` on each child that should be set on this element.  I'd also say that would be better done in CSS

Reply:
> This seems unnecessary; it's overridden on every single child. I'd say rather than setting `fill=currentColor` on each child that should be set on this element. I'd also say that would be better done in CSS

https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/492e2a6f81dd3ad4c95833298a63274b7eeb52fd